public/app/features/search/service/unified.ts

```diff
@@ -273,14 +275,15 @@ function toDashboardResults(hits: SearchHit[]): DataFrame {
   }
   const dashboardHits = hits.map((hit) => {
     let location = hit.folder;
-    if (hit.kind === 'dashboard' && isEmpty(location)) {
+    if (hit.resource === 'dashboards' && isEmpty(location)) {
       location = 'general';
     }
 
     return {
       ...hit,
+      url: toURL(hit.resource, hit.name),
       location,
-      name: hit.title,
+      name: hit.title, // 🤯 FIXME hit.name is k8s name, eg grafana dashboards UID
```

Comment:
Once we have things working more smoothly, we will likely want to change anywhere we use "name" to "title" so we can avoid the guaranteed confusion with uid/name